apps/directory/src/components/partial-views/ApplicationHeaderPartialView.vue

```diff
@@ -7,6 +7,14 @@
         </div>
       </div>
       <div class="col-4 d-flex justify-content-end">
+        <router-link
```

Comment:
I now have to click 3 times before the settings dialog opens. The second time there is a reload.

Reply:
somehow the lazy loading doesnt work for this screen. So removed that function. now one click again! (when logged in ;) )